From 305727ceda9abc8c86174e503889887792a2598e Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Mon, 20 Mar 2023 17:12:02 +0530
Subject: [PATCH] PRODUCT_SUPPORTS_VERITY is obsolete

This resolves the error:
error: PRODUCT_SUPPORTS_VERITY is obsolete. VB 1.0 and related variables are no longer supported.

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/kernel/AndroidBoard.mk     | 9 ---------
 groups/trusty/true/trusty_efi.mk  | 2 --
 groups/trusty/true/trusty_vsbl.mk | 3 ---
 3 files changed, 14 deletions(-)

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index 0c4225e..be8f401 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -225,15 +225,6 @@ $(PRODUCT_OUT)/ramdisk.img: $(LOCAL_KERNEL_PATH)/copy_modules
 endif
 {{/camera_cos_hack}}
 
-{{#slot-ab}}
-ifeq ($(PRODUCT_SUPPORTS_VERITY), true)
-DM_VERITY_CERT := $(LOCAL_KERNEL_PATH)/verity.x509
-$(DM_VERITY_CERT): $(PRODUCTS.$(INTERNAL_PRODUCT).PRODUCT_VERITY_SIGNING_KEY).x509.pem $(OPENSSL)
-	$(transform-pem-cert-to-der-cert)
-$(LOCAL_KERNEL): $(DM_VERITY_CERT)
-endif
-
-{{/slot-ab}}
 $(LOCAL_KERNEL): $(MINIGZIP) $(KERNEL_CONFIG) $(BOARD_DTB) $(KERNEL_DEPS)
 	$(KERNEL_MAKE_CMD) $(KERNEL_MAKE_OPTIONS)
 	$(KERNEL_MAKE_CMD) $(KERNEL_MAKE_OPTIONS) modules
diff --git a/groups/trusty/true/trusty_efi.mk b/groups/trusty/true/trusty_efi.mk
index 1d0a3b3..2367dfb 100644
--- a/groups/trusty/true/trusty_efi.mk
+++ b/groups/trusty/true/trusty_efi.mk
@@ -1,7 +1,5 @@
 INSTALLED_TOS_IMAGE_TARGET := $(PRODUCT_OUT)/tos.img
 INSTALLED_MULTIBOOT_IMAGE_TARGET := none
-TOS_SIGNING_KEY := $(PRODUCT_VERITY_SIGNING_KEY).pk8
-TOS_SIGNING_CERT := $(PRODUCT_VERITY_SIGNING_KEY).x509.pem
 
 tosimage: $(INSTALLED_TOS_IMAGE_TARGET)
 
diff --git a/groups/trusty/true/trusty_vsbl.mk b/groups/trusty/true/trusty_vsbl.mk
index 9c63b90..56c5723 100644
--- a/groups/trusty/true/trusty_vsbl.mk
+++ b/groups/trusty/true/trusty_vsbl.mk
@@ -1,6 +1,3 @@
-SIGNING_KEY := $(PRODUCT_VERITY_SIGNING_KEY).pk8
-SIGNING_CERT := $(PRODUCT_VERITY_SIGNING_KEY).x509.pem
-
 INSTALLED_TOS_IMAGE_TARGET := $(PRODUCT_OUT)/tos.img
 
 tosimage: $(INSTALLED_TOS_IMAGE_TARGET)
-- 
2.17.1

